person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    is_of_age? || @parent_permission
+  end
```

Comment:
- [ ] you are calling the method `is_of_age?` however the actual method you should be calling is called `of_age?` instead.